package.json

```diff
@@ -0,0 +1,99 @@
+{
+    "name": "web-form-builder",
+    "version": "1.0.0-alpha.1",
+    "description": "A module for the web form builder",
+    "main": "dist/cjs/index.js",
+    "module": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "scripts": {
+        "build": "npm run rollup && npm run post:build",
```

Comment:
we still need the post:build to copy over the package.json and readme

Reply:
Oh for this repo since there's only the root module, we don't need the post build method. The DS has it because it wanted to support sub-directory imports on the components.

Reply:
whoops not aware of this, then it's not an issue